HarfangLab/manifest.json

```
@@ -26,7 +26,7 @@
   "name": "HarfangLab",
   "uuid": "8380240b-61a4-48b7-93e4-044a7ee2309b",
   "slug": "harfanglab",
-  "version": "1.23.1",
+  "version": "1.24",
```

We follow the [semver](https://semver.org/) convention for version number.
```suggestion
  "version": "1.24.0",
```